### PR TITLE
Add set_variables and unit completion commands

### PR DESCRIPTION
**Pull request by 1fxe**
- Fixed small bug with `title` not working on linux
- Renamed instances on `map` in `command.py` to `current_map` as map is a builtin function
- added set_variables command
- add unit completion
- fix version migration due to `null` version in `intial.json`
- add cash reward on level up

Also awesome project ❤️

### CODE REVIEW

File: command.py [hunk 1]
```diff
@@ -472,15 +472,15 @@ def do_command(USERID, cmd, args):
         collected_at_timestamp = timestamp_now()
         level = 0 # TODO 
         orientation = 0
-        map["items"] += [[id, x, y, orientation, collected_at_timestamp, level]]
+        # current_map["items"] += [[id, x, y, orientation, collected_at_timestamp, level]]
```
Comment (1fxe) [inline command.py:475]: woops I forgot about commenting this

File: command.py [hunk 2]
```diff
@@ -531,6 +541,21 @@ def do_command(USERID, cmd, args):
 
         print(f"Ended quest {quest_id}.")
 
+    elif cmd == Constant.CMD_BUY_STORED_ITEM_CASH:
```
Comment (1fxe) [inline command.py:544]: image

**Comment by 1fxe** [comment]
I will look into https://github.com/AcidCaos/socialemperors/pull/15 and https://github.com/AcidCaos/socialemperors/pull/12 he ran into some issues with file but did add some commands

**Comment by Nerroth** [comment]
Nice work!

Though I'm unsure if we can merge all changes to the codebase, since later when we will have the time to, the plan is to rewrite most of the code so it is based off of the codebase for Social Wars and things in the code are implemented properly. The code here requires lots of refactoring so it would be best to do that before fixing things.

Also it would be great if you joined our [Discord](https://discord.gg/zW5gSbQJBw) so we can reach out to you more easily, if you're already there just ping me.

Thanks!

**Comment by AcidCaos** [comment]
Cool!

Partially pulled #21 :
- Fixed small bug with `title` not working on linux
- fix version migration due to `null` version in `intial.json`

To review after refactoring (changes at `command.py`):
- Renamed instances on `map` in `command.py` to `current_map` as map is a builtin function
- added set_variables command
- add unit completion
- add cash reward on level up

**Pull request closed without merging**